src/core/border_router/routing_manager.hpp

```diff
@@ -285,13 +286,26 @@ class RoutingManager : public InstanceLocator
      * The prefix is extracted from the platform generated RA messages handled by `ProcessPlatfromGeneratedNd()`.
      *
      * @param[out] aPrefixInfo      A reference to where the prefix info will be output to.
-     *
      * @retval kErrorNone           Successfully retrieved the OMR prefix.
      * @retval kErrorNotFound       There are no valid PD prefix on this BR.
      * @retval kErrorInvalidState   The Border Routing Manager is not initialized yet.
      *
      */
     Error GetPdOmrPrefix(PrefixTableEntry &aPrefixInfo) const;
+
+    /**
+     * Returns platform generated RA message processed information.
+     *
+     * The group of data is extracted from 'GetPdProcessedRaInfo()`.
```

Comment:
This is confusing. The method itself is `GetPdProcessedRaInfo`. In comments of method, the first line is a brief summary. Then after a blank, there should be some explanatio and description. If there isn't anything that worth special explanation, we can skip the explanation.